### PR TITLE
[plugin.video.longnow] 0.1.0

### PR DESCRIPTION
**Pull request by desperadoduck**
A small plugin to view the seminar videos of the longnow foundation.

### CODE REVIEW

File: plugin.video.longnow/addon.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                            my_addon.getLocalizedString(clo.INF_MSG_SERVER) % "".join(errorblock.findAll(text=True)))
+        return
+                       
+    hdstring=re.compile('Full HD Video');
+    for dl in listSoup.findAll("ul", 'download_list'):
+        hdlink=dl.find('a',title=hdstring);
+        if hdlink is not None:
+            name=dl.parent.parent.find('td',{'class':'title'}).a.string
+            url=hdlink['href'];
+            li = xbmcgui.ListItem(name, iconImage='DefaultVideo.png')
```
Comment (learningit) [inline plugin.video.longnow/addon.py]: The inits for iconImage and thumb in listitem are deprecated. Please use listitem.setArt() instead

**Comment by learningit** [comment]
The rest looks ok. Please make the listitem init change, then I can merge. Thanks.

**Comment by desperadoduck** [comment]
Ok, thanks! I'll try to fix this on the weekend.

**Comment by desperadoduck** [comment]
OK, I updated the pull request. i hope thats ok, otherwise I file a new one. Apart from using setArt, I made the following small fixes: 
* fix dialogs if no user or password are given
* change plugin id to match name of the plugin directory
* update version
* explictly require plugin version for dependencies in addon.xml

**Comment by learningit** [comment]
Please squash your commits into a single commit and name that commit in the same format as the PR. Please see: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

for how-to.

**Comment by desperadoduck** [comment]
Ok, thanks, did that.

File: plugin.video.longnow/addon.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.longnow" name="Long Now Seminars" version="0.0.4" provider-name="desperadoduck">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
+    <import addon="script.module.beautifulsoup" version="3.2.1" />
```
Comment (learningit) [inline plugin.video.longnow/addon.xml]: Sorry I missed this earlier xbmc.python should be set to 2.24.0 for Jarvis.

**Comment by desperadoduck** [comment]
allright, I changed the required version of xbmc.python to 2.24.0.

**Pull request merged**